### PR TITLE
Add cross-org visibility doc

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do? What is the motivation?
Adds a new document for cross-organization visibility.
See https://datadoghq.atlassian.net/browse/DOCS-6300
Product Management requested that while the feature is in private beta, the page does not appear in side navigation and is not indexed in internal or external search.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by urseberry** [comment]
https://datadoghq.atlassian.net/browse/DOCS-6892

File: content/en/account_management/org_settings/cross_org_visibility.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+### In the API
+
+You can define cross-organization queries in the following endpoint:
+- [Timeseries][8]
+
+When you define dashboard widgets in the dashboard API, use the `cross_org_uuids` parameter in the JSON widget definition payload to identify the source organization in a cross-organization query.
+
+The `cross_org_uuids` parameter is optional. If you omit `cross_org_uuids`, the query runs on the same organization in which you defined the widget.
+
+The [Datadog Terraform Provider][9] does not support cross-organization queries.
```
Comment (estherk15) [inline content/en/account_management/org_settings/cross_org_visibility.md:123]: Depending on how popular this is, consider making this an alert and move to the top of the section?
```suggestion
<div class="alert alert-info">
The <a href="https://registry.terraform.io/providers/DataDog/datadog/latest/docs">Datadog Terraform Provider</a> does not support cross-organization queries.
</div>
```

**Review by estherk15 — APPROVED** [review]
Wow, I think is a very long awaited feature! Very cool!

File: content/en/account_management/org_settings/cross_org_visibility.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+{{< callout url="#" header="false" btn_hidden="true">}}
+  Cross-organization visibility is in <b>private beta</b> for customers with Enterprise plans. If you're interested in the feature, reach out to your Technical Account Manager or Customer Success Manager.
+{{< /callout >}} 
+
+
+## Overview
+
+Some companies rely on multiple Datadog [organizations][1]  to isolate data for compliance or other reasons.
+
+Cross-organization visibility allows customers to share data between different organizations in the same account, and thus show insights from multiple organizations in one place. Cross-organization visibility works with the **Dashboard** and **Notebook** products.
```
[the remark this answers is not in the document]
Comment (urseberry) [reply content/en/account_management/org_settings/cross_org_visibility.md]: good catch! It's Cross-org. I'll update the title.

File: content/en/account_management/org_settings/cross_org_visibility.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+## Create a widget with cross-organization data
+
+Cross-organization [dashboard widgets][2] are available for Datadog organizations that exist as a _destination_ organization for at least one [connection](#configure-connections).
+
+Each query in a widget can display data from a single organization. You can combine queries into a cross-organization formula query.
+
+### In the UI
+
+Dashboard widgets allow you to create cross-organization queries when the following conditions are met:
```
[the remark this answers is not in the document]
Comment (urseberry) [reply content/en/account_management/org_settings/cross_org_visibility.md]: It includes Notebooks too. Thanks for catching this!

**Pull request merged**